src/NServiceBus.Hosting.Windows/StartableAndStoppable/StartableAndStoppableRunner.cs

```diff
@@ -58,9 +65,13 @@ public Task Start(IMessageSession session)
             {
                 try
                 {
-                    var task = stoppable.Stop(session).ThrowIfNull();
-
                     var stoppable1 = stoppable;
+                    var stoppableName = stoppable1.GetType().AssemblyQualifiedName;
+
+                    var longRunningTokenSource =
+                        LongRunningWarning($"The stop method for {stoppableName} is taking too long to complete. The endpoint will not shutdown until this operation has completed.");
```

Comment:
Same comment as the other message, change to "is taking a long time to complete"

Also, it should be "shut down" instead of "shutdown".